contrib/plugins/funds/funds.py

```diff
@@ -0,0 +1,100 @@
+#!/usr/bin/env python3
+""" This plugin gives you a nicer overview of the funds that you own.
+
+Instead of calling listfunds and adding all outputs and channels
+this plugin does that for you.
+
+Activate the plugin with: 
+`lightningd --plugin=PATH/TO/LIGHTNING/contrib/plugins/funds/funds.py`
+
+Call the plugin with: 
+`lightning-cli funds`
+
+
+Author: Rene Pickhardt (https://ln.rene-pickhardt.de)
+Development of the plugin was sponsored by fulmo.org
+"""
+
+import json
+
+from lightning.lightning import LightningRpc
+from lightning.plugin import Plugin
+from os.path import join
+
+rpc_interface = None
+plugin = Plugin(autopatch=True)
+
+
+@plugin.method("funds")
+def funds(plugin, unit="s"):
+    """Lists the total funds the lightning node owns off- and onchain in {unit}.
+
+{unit} can take the following values:
+s, satoshi, satoshis to depict satoshis
+b, bit, bits to depict bits
+m, milli, btc to depict milliBitcoin
+B, bitcoin, btc to depict Bitcoins
+
+When not using Satoshis (default) the comma values are rounded off."""
+
+    switcher = {
+        "bitcoin": "BTC",
+        "btc": "BTC",
+        "satoshi": "sat",
+        "satoshis": "sat",
+        "bit": "bit",
+        "bits": "bit",
+        "milli": "mBTC",
+        "mbtc": "mBTC",
+        "millibtc": "mBTC",
+        "B": "BTC",
+        "s": "sat",
+        "m": "mBTC",
+        "b": "bit",
+    }
+
+    if unit != "B":
```

Comment:
Why is `B` being special-cased here? imho it'd still work using the above mapping.

Reply:
I had a to lower of the argument before checking the switch statement. however `B` is for BTC and `b` for bits. I fully agree it is ugly but I did not have any better idea and I thought about this several times for more than a couple of minutes. Totally open to a better suggestion here

Reply:
Oh I see, sorry for missing that. I can't think of an elegant way, maybe something will pop up :-)

Reply:
don't want to doubt you. But I doubt it. I was very unhappy about this and thought about it several times :( but maybe it is just ok (: